### PR TITLE
Shifting arithmatic fix

### PR DESCRIPTION
**Pull request by hassan-ali-lab**
Sir, I have reviewed array, backtracking and bit related README.md files very consciously and found some issues.

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+vcs.xml
+modules.xml
+go-dsa.iml
```
Comment (spring1843) [inline .idea/.gitignore:11]: Can we remove the extra ones from here? I think only the last line is relevant to this project.

File: bit/README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ func printBinary(n int) {
 
 ## Arithmetic by Shifting
 
-Left shifting can be viewed as a division operation by 2 raised to the power of a specified number, while right shifting can be viewed as a multiplication operation by 2 raised to the power of a specified number. For instance, a << b can be interpreted as dividing a by 2^b, and a >> b can be interpreted as multiplying a by 2^b.
+Left shifting can be viewed as a division operation by 2 raised to the power of a specified number, while right shifting can be viewed as a multiplication operation by 2 raised to the power of a specified number. For instance, a << b can be interpreted as multiplying a by 2^b, and a >> b can be interpreted as dividing a by 2^b.
```
Comment (spring1843) [inline bit/README.md:48]: Nice catch

**Review by spring1843 — CHANGES_REQUESTED** [review]

File: bit/README.md (hunk 2 above)
Comment (spring1843) [reply bit/README.md:48]: Should the first sentence of the paragraph change as well? e.g. Right shifting can be viewed as multiplication.... ?

Comment (hassan-ali-lab) [reply bit/README.md:48]: yes, the first sentence of the paragraph should also be changed.

**Review by hassan-ali-lab — COMMENTED** [review]
fixed error regrading **Arithmetic by Shifting** in bit/README.md

**Review by spring1843 — APPROVED** [review]

**Review by spring1843 — APPROVED** [review]
LGTM thank you for your contribution

**Pull request merged**